saas/api/metrics.py

```diff
@@ -281,8 +303,8 @@ class RevenueMetricAPIView(DateRangeContextMixin, ProviderMixin,
     def get(self, request, *args, **kwargs):
         #pylint:disable=unused-argument
         dates = convert_dates_to_utc(
-            month_periods(12, self.ends_at, tz=self.timezone))
-
+            self.period_func(12, from_date=self.ends_at, tz=self.timezone)
```

Comment:
12 months period is a year. It seems natural. But 12 weeks or 12 days seems odd. Maybe the number of periods should be defined as a query parameter and default to a sensible value based on the period (ex: 7 for days).

Reply:
You want the metrics page to show the more natural periods as well? Or keep 12 there and only make the changes to the API view while adding in a "number of period" parameter that is passed in as 12 from the metrics page? 

Wasn't sure how changing the number of periods affected Djaoapp or any other existing usages if I changed the number of periods showing on the metrics page